README_images.md

```diff
@@ -0,0 +1,405 @@
+
+# cob4 Installation automatically with USB/CD/DVD
```

Comment:
please remove this file, this is an old version of https://github.com/ipa320/setup_cob4/blob/master/images_config/README_images.md